src/applications/personalization/dashboard/tests/components/IconCTALink.unit.spec.jsx

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
```

Comment:
Can we render the component using the react testing library instead of enzyme in this test? Reason that I mention it is that I know we are looking to move away from Enzyme and it may not be supported in future versions of React if I remember correctly.